cmd/influx_stress/influx_stress.go

```diff
@@ -29,32 +31,36 @@ func main() {
 		defer pprof.StopCPUProfile()
 	}
 
-	c, err := stress.NewConfig(*config)
-	if err != nil {
-		fmt.Println(err)
-		return
-	}
+	if *useV2 {
+		v2.RunStress(*config)
+	} else {
```

Comment:
Remove this `else`, add a return under `v2.RunStress(*config)`, and move all the else code to the outer code-level.